### PR TITLE
add chain id for web3q galileo

### PR DESCRIPTION
**Pull request by ping-ke**

### CODE REVIEW

File: _data/chains/eip155-3334.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "W3Q",
+    "decimals": 18
+  },
+  "infoURL": "https://galileo.web3q.io/home.w3q/",
+  "shortName": "w3q-g",
+  "chainId": 3334,
+  "networkId": 3334,
+  "explorers": [{
+    "name": "w3q-galileo",
+    "url": "https://explorer.galileo.web3q.io/",
```
Comment (ligi) [inline _data/chains/eip155-3334.json:20]:
```suggestion
    "url": "https://explorer.galileo.web3q.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please remove the slash

**Comment by ping-ke** [comment]
> Please remove the slash
Slash has been removed.

**Comment by ping-ke** [comment]
> Please remove the slash
Slash has been removed.

**Pull request merged**